### ISSUE TITLE
Document minimum React Native version supported

### ISSUE THREAD

**Issue by chrisbobbe** [charpeni/react-native-url-polyfill#154]
Thanks for this! 🙂 The [list of bugs](https://github.com/charpeni/react-native-url-polyfill#why-do-we-need-this) in React Native core's polyfill in the README is quite startling, and I've reproduced https://github.com/facebook/react-native/issues/25717, so far.

-----

[A file](https://github.com/charpeni/react-native-url-polyfill/blob/master/js/URL.js) in `react-native-url-polyfill`, which was [added](https://github.com/charpeni/react-native-url-polyfill/commit/37bb848a0c8ef60b29df12de825349f0824304d7#diff-375a2b7b7d929526f0dd58cd8bb8665b) in 1.0.0, [the initial release](https://github.com/charpeni/react-native-url-polyfill/releases/tag/v1.0.0), depends on [a file](https://github.com/facebook/react-native/blob/master/Libraries/Blob/NativeBlobModule.js) in React Native, which [was added](https://github.com/facebook/react-native/commit/342c81d75448e760ae32d62ebd9004bf6eeeff46) in v0.61.0.

On RN v0.59.10, which I'm using, I get this error from Metro:

```
error: bundling failed: Error: Unable to resolve module `react-native/Libraries/Blob/NativeBlobModule` from `/Users/chrisbobbe/dev/zulip-mobile/node_modules/react-native-url-polyfill/js/URL.js`: Module `react-native/Libraries/Blob/NativeBlobModule` does not exist in the Haste module map
```

There may be other factors preventing `react-native-url-polyfill` from working prior to RN v0.61.0, or indeed it might continue to not work until after that (maybe even v0.62?).

It would be nice to have a note, maybe in the install instructions in the README, on what the earliest supported React Native version is, for the latest release of `react-native-url-polyfill`, and perhaps mention in the notes for each release if that number changes.

Hopefully this data point will help with that. 🙂

**Comment by charpeni** [comment]
Thanks for this detailed issue, I appreciate the feedback!

I’ll make sure to mention it clearly in the README, but also specifying supported versions of React Native in `peerDependencies` so we can have a warning on installation too.

What you mentioned regarding upcoming releases is also important to prevent breaking changes. I’ll make sure to keep the tests up-to-date with it.

I didn’t had a lot of time for this recently, but I should be able to find some soon.

Thanks again for filling this!

**Comment by chrisbobbe** [comment]
Great, thanks so much! Stay safe, with COVID-19, and all.

**Comment by charpeni** [comment]
I'm currently working on adding support for older version (0.59) and also making sure to test a larger band of React Native versions.

**Comment by charpeni** [comment]
@chrisbobbe [`react-native-url-polyfill@1.2.0-rc.0`](https://www.npmjs.com/package/react-native-url-polyfill/v/1.2.0-rc.0) is now available with backward support for RN 0.60 & 0.59, and probably more!

Let me know how it goes for you so I can release it officially.

**Comment by chrisbobbe** [comment]
Confirmed that 1.2.0-rc.0 worked for me on `react-native` v0.59.10 and on `react-native` v0.60.6! Thank you!
